fix: setuptools.py

hanxiao opened the pull request.

hanxiao committed:
fix: setuptools.py

hanxiao committed:
Update setup.py

alexcg1 commented on setup.py:
```diff
@@ -1,17 +1,54 @@
+import sys
+
+if sys.version_info < (3, 7, 0):
+    raise OSError(f'Jina requires Python 3.7 and above, but yours is {sys.version}')
```
We shouldn't worry about this, since we only use this as a front-end. No Jina code is involved. So we should tie it to whatever version of Streamlit is important

alexcg1 commented on setup.py:
```diff
@@ -1,17 +1,54 @@
     version="0.1.1",
-    author="Jina AI",
+    author='Jina Dev Team',
     author_email="alex.cg@streamlit.io",
```
Crap, I need to change this to alex.cg@jina.ai

alexcg1 committed:
Update setup.py

- Set minimum Python version to what *streamlit* needs, not Jina. Since this is just frontend and doesn't rely on Jina backend
- Fix my email
- Add more keywords

alexcg1 commented:
I've changed the PR to take into account my feedback and will merge now
